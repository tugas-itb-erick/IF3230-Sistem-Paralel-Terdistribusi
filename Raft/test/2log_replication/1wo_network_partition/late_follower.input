5
set_timeout 3 0
simulate 10
client_send_request 3 + 1
simulate 10
get_state 1
crash 1
client_send_request 3 - 2
simulate 10
get_state 2
crash 2
client_send_request 3 * 3
simulate 10
get_state 4
crash 4
client_send_request 3 || 1
simulate 10
get_state 5
crash 5
client_send_request 3 + 7
simulate 10
restart 1
restart 2
restart 4
restart 5
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
simulate 20
get_state 1
get_state 2
get_state 3
get_state 4
get_state 5
end_simulation