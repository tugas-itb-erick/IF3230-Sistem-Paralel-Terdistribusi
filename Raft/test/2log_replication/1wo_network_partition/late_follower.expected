S1 follower term:2 voted_for:3 commit_index:1 data:1 logs:T0[0] T2[+1]
S2 follower term:2 voted_for:3 commit_index:2 data:-1 logs:T0[0] T2[+1-2]
S4 follower term:2 voted_for:3 commit_index:3 data:-3 logs:T0[0] T2[+1-2*3]
S5 follower term:2 voted_for:3 commit_index:3 data:-3 logs:T0[0] T2[+1-2*3||1]
S1 follower term:2 voted_for:-1 commit_index:1 data:1 logs:T0[0] T2[+1]
S2 follower term:2 voted_for:-1 commit_index:2 data:-1 logs:T0[0] T2[+1-2]
S3 leader term:2 voted_for:3 commit_index:3 data:-3 logs:T0[0] T2[+1-2*3||1+7]
S4 follower term:2 voted_for:-1 commit_index:3 data:-3 logs:T0[0] T2[+1-2*3]
S5 follower term:2 voted_for:-1 commit_index:3 data:-3 logs:T0[0] T2[+1-2*3||1]
S1 follower term:2 voted_for:-1 commit_index:5 data:8 logs:T0[0] T2[+1-2*3||1+7]
S2 follower term:2 voted_for:-1 commit_index:5 data:8 logs:T0[0] T2[+1-2*3||1+7]
S3 leader term:2 voted_for:3 commit_index:5 data:8 logs:T0[0] T2[+1-2*3||1+7]
S4 follower term:2 voted_for:-1 commit_index:5 data:8 logs:T0[0] T2[+1-2*3||1+7]
S5 follower term:2 voted_for:-1 commit_index:5 data:8 logs:T0[0] T2[+1-2*3||1+7]
